bootcamp/tutorials/integration/llama_agents_metadata.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>this piece is the jewel of the story, which shows how to use an agent to extract metadata filter and apply that during question answering. so</p><p>1) this point can be mentioned at the very begin of this story.</p><p>2) this part is worth a subtitle or some kind of highlighting</p><p> </p><p>What could make this story even more shiny is to show an example that, without the auto-metadata-filtering, the question answering will confuse itself by answering a qusetion about lyft with information about uber. (if that's possible to find such a counter case)</p><p> </p><p>The other case above does show different result, but it's unclear to the reader whether it's wrong and why:</p><p> </p><p>Response without metadata filtering: </p><p>The total revenue for Uber in 2021 was $17,455 million. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1401/discussion/#comment-1705463994'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>